### ISSUE TITLE
Non-text Content 1.1.1 - WAI-ARIA image is missing alternative text

### ISSUE THREAD

**Issue by atsea** [UDelIT/udthemebrand#39]
Suggested fix:

- Add to header img `aria-label="Go to the University of Delaware home page."`
- Add to footer img `aria-label="Go to the University of Delaware home page."`

[Site Improve Page Report](https://my2.siteimprove.com/Inspector/404526/Accessibility/Page?pageId=32809310644&impmd=02D94A00E6B60C5C1F421DC34C09CE08&level=a#/Criterion/1.1.1/Check/111)

**Comment by atsea** [comment]
Fix says to change from `role="img"` to `role="presentation`. This will cause screen reader to ignore. Trying this first for test.

**Comment by atsea** [comment]
After much research role="img" is the best solution. This is a manual fix based on accessibility testing. Differentiated text as best I can.
